eslint-plugin-local-rules/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "eslint-plugin-local-rules",
+  "version": "1.0.0",
```

Comment:
Let's add `"private": true` to make sure this never gets published, then we don't need `"version"` field.

Reply:
Replaced the custom implementation with `no-restricted-syntax`